src/styles/variables/miscellaneous.css

```diff
@@ -1,10 +1,15 @@
 /*
-  Variables defined in Figma, but not easily
-  exportable via scripting. The naming of
-  variables is a manual process, but they
-  should align with the name in Figma.
+  Variables defined in Figma, but they're Figma effects, not easily exportable via scripting,
+  or Figma doesn't support similar syntax to CSS. The naming of these variables is a manual
+  process. They should align with the naming format in Figma.
+
+  --glide-core-{collection}-{category*}-{scope*}-{property}-{variant*}--{state*}
+  * = optional
 */
 :root {
+  --glide-core-typography-family-primary: 'Nunito', system-ui;
+  --glide-core-typography-family-monospace:
+    'Oxygen Mono', ui-monospace, monospace;
```

Comment:
Manually moved these here. No changes for consumers though, as our build merges all stylesheets.